packaging/linux/run_keybase

```diff
@@ -157,8 +157,11 @@ run_redirector() {
 
   # Only start the root redirector if it hasn't been explicitly disabled.
   if [ "$disable" != "true" ]; then
-    logdir="${XDG_CACHE_HOME:-$HOME/.cache}/keybase"
-    nohup keybase-redirector /keybase >> "$logdir/keybase.redirector.log" 2>&1 &
+    # There's no good place to put logs as a user, since this binary
+    # will suid to root and be unable to write to ~/.cache/keybase.
```

Comment:
Root can't write to a user homedir? That's surprising to me. (I would've predicted the opposite problem, that the write succeeds but sometimes creates files/dirs owned by root that should've been owned by the user.)